docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -20,8 +20,8 @@
 |spring.cloud.gateway.filter.hystrix.enabled | `+++true+++` | Enables the hystrix filter.
 |spring.cloud.gateway.filter.json-to-grpc.enabled | `+++true+++` | Enables the JSON to gRPC filter.
 |spring.cloud.gateway.filter.local-response-cache.enabled | `+++true+++` | Enables the local-response-cache filter.
-|spring.cloud.gateway.filter.local-response-cache.size |  | 
-|spring.cloud.gateway.filter.local-response-cache.time-to-live |  | 
+|spring.cloud.gateway.filter.local-response-cache.size |  | Maximum size of the cache to evict entries for this route (in KB, MB and GB)
+|spring.cloud.gateway.filter.local-response-cache.timeToLive |  | Time to expire a cache entry (expressed in s for seconds, m for minutes, and h for hours)
```

Comment:
This file is auto generated and should be edited either in the config props class or in the addition config metadata json file

Reply:
Done on the config metadata file: https://github.com/spring-cloud/spring-cloud-gateway/pull/2848/commits/7a684d77b4fdd2818014611d332d5b95165a7311